eservice-descriptors-archiver/src/main/scala/it/pagopa/interop/eservicedescriptorsarchiver/ApplicationConfiguration.scala

```diff
@@ -7,11 +7,11 @@ object ApplicationConfiguration {
 
   private val config: Config = ConfigFactory.load()
 
-  val archivingPurposesQueueUrl: String =
-    config.getString("eservice-versions-archiver.queue.archiving-purposes-queue-url")
-  val maxNumberOfMessagesPerFile: Int   =
-    config.getInt("eservice-versions-archiver.queue.max-number-of-messages-per-file")
-  val visibilityTimeout: Int = config.getInt("eservice-versions-archiver.queue.visibility-timeout-in-seconds")
+  val archivingEservicesQueueUrl: String =
+    config.getString("eservice-descriptors-archiver.queue.archiving-eservices-queue-url")
+  val maxNumberOfMessagesPerFile: Int    =
+    config.getInt("eservice-descriptors-archiver.queue.max-number-of-messages-per-file")
```

Comment:
this config must be removed because it does not exist anymore in the config file